### DISCUSSION TITLE
how to generate the html files to display the results statically?

### DISCUSSION THREAD

**Discussion by isshwar** [rabbitmq/rabbitmq-perf-test#428] [General]
Hi,

I am just trying to run the test from CI/CD pipeline. My pipeline would include to run perftestMulti after every deployment and display the results in html. While the tests could run using perftestMulit, wanted to know if the results can be generated in html for later perusal.I could only see the results generated in js and later some additional steps to view the html graphs.

I could not find any documentation on this? Any information would be really helpful.

Thanks

**Comment by acogoluegnes** [comment]
PerfTestMulti generates a JSON result file only. The extension provides some JavaScript and HTML files to display the results though. This is documented in the [extension readme file](https://github.com/rabbitmq/rabbitmq-perf-test/blob/main/html/README.md).
